web/pandas_web.py

```diff
@@ -363,6 +363,8 @@ def get_source_files(source_path: str) -> typing.Generator[str, None, None]:
     Generate the list of files present in the source directory.
     """
     for root, dirs, fnames in os.walk(source_path):
+        if root.startswith("pandas/pdeps"):
```

Comment:
This could be generalized by allowing a directory in the ignore field in `config.yml`